Source/JavaScriptCore/dfg/DFGByteCodeParser.cpp

```diff
@@ -7992,12 +8009,263 @@ void ByteCodeParser::parseBlock(unsigned limit)
             case UnresolvedPropertyWithVarInjectionChecks:
             case ModuleVar:
             case Dynamic:
+                if (Options::useDebugLog()) dataLogLn("DFG ------------------------------------------------  14.5");
                 RELEASE_ASSERT_NOT_REACHED();
                 break;
             }
             NEXT_OPCODE(op_get_from_scope);
         }
 
+        case op_resolve_and_get_from_scope: {
```

Comment:
let's try to share code with resolve_scope and get_from_scope

Reply:
Updated